vector/src/main/res/layout/item_suggested_room.xml

```diff
@@ -102,6 +102,17 @@
         app:barrierDirection="bottom"
         app:constraint_referenced_ids="roomAvatarBottomSpace" />
 
+    <TextView
```

Comment:
Please declare a style for this TextView

Reply:
I used Body, looks like the correct choice

Reply:
thanks